.circleci/config.yml

```diff
@@ -18,32 +39,31 @@ jobs:
           name: Validate Terraform Formatting
           command: "[ -z \"$(terraform fmt -write=false)\" ] || { terraform fmt -write=false -diff; exit 1; }"
 
-  build:
-    <<: *defaults
-    steps:
-      - checkout
-      - attach_workspace:
-          at: *workspace_root
-      - restore_cache:
-          keys:
-            - dep-{{ checksum "test/Gopkg.lock" }}
```

Comment:
Hm, I understand removing the workspace thing, but removing the cache will make this build slower, right? It'll have to do `dep ensure` 100% from scratch each time?

Reply:
Good point - I've put just the caching back and it's cutting 5 minutes off the total time.